### ISSUE TITLE
Post-processor failed: unexpected EOF

### ISSUE THREAD

**Issue by ubailb1** [mitchellh/packer#2656]
I am building with VMWare vmx post-processor and getting error when sending vmx file to vsphere.  Any advice is appreciated.

2015/08/21 22:10:49 ui: ==> vmware-vmx: Running post-processor: vsphere
==> vmware-vmx: Running post-processor: vsphere
2015/08/21 22:10:49 ui:     vmware-vmx (vsphere): Uploading output-vmware-vmx/packer-vmware-vmx-{{timestamp}}.vmx to vSphere
2015/08/21 22:10:49 packer-post-processor-vsphere: 2015/08/21 22:10:49 Starting ovftool with parameters: --noSSLVerify=false --acceptAllEulas --name=Mac OSX 10.10.5 --datastore=ESXI HDD 12 --diskMode=thin --network=VM_Network --vmFolder=  output-vmware-vmx/packer-vmware-vmx-{{timestamp}}.vmx vi://root:Administr%40tor@10.25.197.12/DevOps/host/Mac Mini Lab
    vmware-vmx (vsphere): Uploading output-vmware-vmx/packer-vmware-vmx-{{timestamp}}.vmx to vSphere
2015/08/21 22:45:43 packer-post-processor-vsphere: runtime: out of memory: cannot allocate 68719476736-byte block (68721573888 in use)
2015/08/21 22:45:43 packer-post-processor-vsphere: fatal error: out of memory
2015/08/21 22:45:43 packer-post-processor-vsphere:
2015/08/21 22:45:43 packer-post-processor-vsphere: runtime stack:
2015/08/21 22:45:43 packer-post-processor-vsphere: runtime.throw(0x67d64a)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/runtime/panic.go:491 +0xad
2015/08/21 22:45:43 packer-post-processor-vsphere: runtime.largeAlloc_m()
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/runtime/malloc.c:390 +0xee
2015/08/21 22:45:43 packer-post-processor-vsphere: runtime.onM(0xc208012000)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/runtime/asm_amd64.s:257 +0x68
2015/08/21 22:45:43 packer-post-processor-vsphere: runtime.mstart()
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/runtime/proc.c:818
2015/08/21 22:45:43 packer-post-processor-vsphere:
2015/08/21 22:45:43 packer-post-processor-vsphere: goroutine 35 [running]:
2015/08/21 22:45:43 packer-post-processor-vsphere: runtime.switchtoM()
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/runtime/asm_amd64.s:198 fp=0xc208093470 sp=0xc208093468
2015/08/21 22:45:43 packer-post-processor-vsphere: runtime.mallocgc(0xffffffe00, 0x328920, 0x1, 0x3c)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/runtime/malloc.go:199 +0x9f3 fp=0xc208093520 sp=0xc208093470
2015/08/21 22:45:43 packer-post-processor-vsphere: runtime.newarray(0x328920, 0xffffffe00, 0xc208021180)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/runtime/malloc.go:365 +0xc1 fp=0xc208093558 sp=0xc208093520
2015/08/21 22:45:43 packer-post-processor-vsphere: runtime.makeslice(0x318700, 0xffffffe00, 0xffffffe00, 0x0, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/runtime/slice.go:32 +0x15c fp=0xc2080935a0 sp=0xc208093558
2015/08/21 22:45:43 packer-post-processor-vsphere: bytes.makeSlice(0xffffffe00, 0x0, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/bytes/buffer.go:191 +0x6a fp=0xc2080935f0 sp=0xc2080935a0
2015/08/21 22:45:43 packer-post-processor-vsphere: bytes.(*Buffer).ReadFrom(0xc20811c2a0, 0x776fc0, 0xc208104040, 0x7fffffc02, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/bytes/buffer.go:163 +0xda fp=0xc208093688 sp=0xc2080935f0
2015/08/21 22:45:43 packer-post-processor-vsphere: io.Copy(0x7716f8, 0xc20811c2a0, 0x776fc0, 0xc208104040, 0x0, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/io/io.go:358 +0x13d fp=0xc208093740 sp=0xc208093688
2015/08/21 22:45:43 packer-post-processor-vsphere: os/exec.func·003(0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/os/exec/exec.go:221 +0x7d fp=0xc2080937a0 sp=0xc208093740
2015/08/21 22:45:43 packer-post-processor-vsphere: os/exec.func·004(0xc2080fe1c0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/os/exec/exec.go:328 +0x2d fp=0xc2080937d8 sp=0xc2080937a0
2015/08/21 22:45:43 packer-post-processor-vsphere: runtime.goexit()
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/runtime/asm_amd64.s:2232 +0x1 fp=0xc2080937e0 sp=0xc2080937d8
2015/08/21 22:45:43 packer-post-processor-vsphere: created by os/exec.(*Cmd).Start
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/os/exec/exec.go:329 +0xb6d
2015/08/21 22:45:43 packer-post-processor-vsphere:
2015/08/21 22:45:43 packer-post-processor-vsphere: goroutine 1 [select, 34 minutes]:
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/hashicorp/yamux.(*Stream).Read(0xc2080ec000, 0xc208119000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/yamux/stream.go:125 +0x3ab
2015/08/21 22:45:43 packer-post-processor-vsphere: bufio.(*Reader).fill(0xc20802e600)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/bufio/bufio.go:97 +0x1ce
2015/08/21 22:45:43 packer-post-processor-vsphere: bufio.(*Reader).ReadByte(0xc20802e600, 0x82, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/bufio/bufio.go:196 +0x86
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/hashicorp/go-msgpack/codec.(*ioDecReader).readn1(0xc20803be90, 0x16)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/go-msgpack/codec/decode.go:90 +0x45
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/hashicorp/go-msgpack/codec.(*msgpackDecDriver).initReadNext(0xc20801f420)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/go-msgpack/codec/msgpack.go:540 +0x48
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/hashicorp/go-msgpack/codec.(*Decoder).decode(0xc20802e660, 0x30cca0, 0xc20801f440)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/go-msgpack/codec/decode.go:635 +0x59
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/hashicorp/go-msgpack/codec.(*Decoder).Decode(0xc20802e660, 0x30cca0, 0xc20801f440, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/go-msgpack/codec/decode.go:630 +0x7f
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/hashicorp/go-msgpack/codec.(*rpcCodec).read(0xc208039cc0, 0x30cca0, 0xc20801f440, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/go-msgpack/codec/rpc.go:88 +0xd9
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/hashicorp/go-msgpack/codec.(*goRpcCodec).ReadRequestHeader(0xc208039cc0, 0xc20801f440, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/go-msgpack/codec/rpc.go:127 +0x61
2015/08/21 22:45:43 packer-post-processor-vsphere: net/rpc.(*Server).readRequestHeader(0xc208039bc0, 0x771bf8, 0xc208039cc0, 0x0, 0x0, 0xc20801f440, 0x0, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/net/rpc/server.go:575 +0x97
2015/08/21 22:45:43 packer-post-processor-vsphere: net/rpc.(*Server).readRequest(0xc208039bc0, 0x771bf8, 0xc208039cc0, 0xc20807c500, 0xc2080fe280, 0x328820, 0x0, 0x0, 0x0, 0x0, ...)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/net/rpc/server.go:542 +0xa2
2015/08/21 22:45:43 packer-post-processor-vsphere: net/rpc.(*Server).ServeCodec(0xc208039bc0, 0x771bf8, 0xc208039cc0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/net/rpc/server.go:461 +0x6d
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/mitchellh/packer/packer/rpc.(*Server).Serve(0xc20801f380)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/mitchellh/packer/packer/rpc/server.go:141 +0x26f
2015/08/21 22:45:43 packer-post-processor-vsphere: main.main()
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/mitchellh/packer/plugin/post-processor-vsphere/main.go:14 +0xc7
2015/08/21 22:45:43 packer-post-processor-vsphere:
2015/08/21 22:45:43 packer-post-processor-vsphere: goroutine 5 [syscall, 40 minutes]:
2015/08/21 22:45:43 packer-post-processor-vsphere: os/signal.loop()
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/os/signal/signal_unix.go:21 +0x1f
2015/08/21 22:45:43 packer-post-processor-vsphere: created by os/signal.init·1
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/os/signal/signal_unix.go:27 +0x35
2015/08/21 22:45:43 packer-post-processor-vsphere:
2015/08/21 22:45:43 packer-post-processor-vsphere: goroutine 7 [chan receive, 40 minutes]:
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/mitchellh/packer/packer/plugin.func·017()
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/mitchellh/packer/packer/plugin/server.go:92 +0x69
2015/08/21 22:45:43 packer-post-processor-vsphere: created by github.com/mitchellh/packer/packer/plugin.Server
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/mitchellh/packer/packer/plugin/server.go:96 +0xcca
2015/08/21 22:45:43 packer-post-processor-vsphere:
2015/08/21 22:45:43 packer-post-processor-vsphere: goroutine 8 [IO wait]:
2015/08/21 22:45:43 packer-post-processor-vsphere: net.(*pollDesc).Wait(0xc2080801b0, 0x72, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/net/fd_poll_runtime.go:84 +0x47
2015/08/21 22:45:43 packer-post-processor-vsphere: net.(*pollDesc).WaitRead(0xc2080801b0, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/net/fd_poll_runtime.go:89 +0x43
2015/08/21 22:45:43 packer-post-processor-vsphere: net.(*netFD).Read(0xc208080150, 0xc20807f000, 0x1000, 0x1000, 0x0, 0x76ff90, 0xc20811e060)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/net/fd_unix.go:242 +0x40f
2015/08/21 22:45:43 packer-post-processor-vsphere: net.(*conn).Read(0xc20802a080, 0xc20807f000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/net/net.go:121 +0xdc
2015/08/21 22:45:43 packer-post-processor-vsphere: bufio.(*Reader).fill(0xc20802e2a0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/bufio/bufio.go:97 +0x1ce
2015/08/21 22:45:43 packer-post-processor-vsphere: bufio.(*Reader).Read(0xc20802e2a0, 0xc2080dc004, 0xc, 0xc, 0xc208011608, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/bufio/bufio.go:174 +0x26c
2015/08/21 22:45:43 packer-post-processor-vsphere: io.ReadAtLeast(0x7716a8, 0xc20802e2a0, 0xc2080dc004, 0xc, 0xc, 0xc, 0x0, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/io/io.go:298 +0xf1
2015/08/21 22:45:43 packer-post-processor-vsphere: io.ReadFull(0x7716a8, 0xc20802e2a0, 0xc2080dc004, 0xc, 0xc, 0xc, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/io/io.go:316 +0x6d
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/hashicorp/yamux.(*Session).recvLoop(0xc20806e000, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/yamux/session.go:381 +0x10d
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/hashicorp/yamux.(*Session).recv(0xc20806e000)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/yamux/session.go:369 +0x28
2015/08/21 22:45:43 packer-post-processor-vsphere: created by github.com/hashicorp/yamux.newSession
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/yamux/session.go:104 +0x499
2015/08/21 22:45:43 packer-post-processor-vsphere:
2015/08/21 22:45:43 packer-post-processor-vsphere: goroutine 9 [select]:
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/hashicorp/yamux.(*Session).send(0xc20806e000)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/yamux/session.go:331 +0x554
2015/08/21 22:45:43 packer-post-processor-vsphere: created by github.com/hashicorp/yamux.newSession
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/yamux/session.go:105 +0x4b3
2015/08/21 22:45:43 packer-post-processor-vsphere:
2015/08/21 22:45:43 packer-post-processor-vsphere: goroutine 10 [select]:
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/hashicorp/yamux.(*Session).keepalive(0xc20806e000)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/yamux/session.go:287 +0x106
2015/08/21 22:45:43 packer-post-processor-vsphere: created by github.com/hashicorp/yamux.newSession
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/yamux/session.go:107 +0x4de
2015/08/21 22:45:43 packer-post-processor-vsphere:
2015/08/21 22:45:43 packer-post-processor-vsphere: goroutine 11 [select, 40 minutes]:
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/hashicorp/yamux.(*Session).AcceptStream(0xc20806e000, 0xc200000000, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/yamux/session.go:191 +0x1a8
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/mitchellh/packer/packer/rpc.(*muxBroker).Run(0xc20801f360)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/mitchellh/packer/packer/rpc/mux_broker.go:126 +0x37
2015/08/21 22:45:43 packer-post-processor-vsphere: created by github.com/mitchellh/packer/packer/rpc.NewServer
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/mitchellh/packer/packer/rpc/server.go:42 +0x7a
2015/08/21 22:45:43 packer-post-processor-vsphere:
2015/08/21 22:45:43 packer-post-processor-vsphere: goroutine 77 [syscall, 34 minutes]:
2015/08/21 22:45:43 packer-post-processor-vsphere: syscall.Syscall6(0x7, 0xb0d9, 0xc2080a13dc, 0x0, 0xc208122120, 0x0, 0x0, 0xc2080fc300, 0x3b01, 0xc2080fc320)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/syscall/asm_darwin_amd64.s:44 +0x5
2015/08/21 22:45:43 packer-post-processor-vsphere: syscall.wait4(0xb0d9, 0xc2080a13dc, 0x0, 0xc208122120, 0x90, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/syscall/zsyscall_darwin_amd64.go:32 +0x79
2015/08/21 22:45:43 packer-post-processor-vsphere: syscall.Wait4(0xb0d9, 0xc2080a1424, 0x0, 0xc208122120, 0x0, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/syscall/syscall_bsd.go:162 +0x60
2015/08/21 22:45:43 packer-post-processor-vsphere: os.(*Process).wait(0xc2080fe4a0, 0x0, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/os/exec_unix.go:22 +0x103
2015/08/21 22:45:43 packer-post-processor-vsphere: os.(*Process).Wait(0xc2080fe4a0, 0xc208066000, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/os/doc.go:45 +0x3a
2015/08/21 22:45:43 packer-post-processor-vsphere: os/exec.(*Cmd).Wait(0xc208072140, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/os/exec/exec.go:364 +0x23c
2015/08/21 22:45:43 packer-post-processor-vsphere: os/exec.(*Cmd).Run(0xc208072140, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/os/exec/exec.go:246 +0x71
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/mitchellh/packer/post-processor/vsphere.(*PostProcessor).PostProcess(0xc208072280, 0x776f00, 0xc208116100, 0x776f48, 0xc208116120, 0x0, 0x0, 0x0, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/mitchellh/packer/post-processor/vsphere/post-processor.go:138 +0x1557
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/mitchellh/packer/packer/rpc.(*PostProcessorServer).PostProcess(0xc20801f3a0, 0x0, 0xc2080dc0b0, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/mitchellh/packer/packer/rpc/post_processor.go:83 +0x30e
2015/08/21 22:45:43 packer-post-processor-vsphere: reflect.Value.call(0x3a1da0, 0x3dde90, 0x13, 0x459370, 0x4, 0xc2080a1f28, 0x3, 0x3, 0x0, 0x0, ...)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/reflect/value.go:419 +0x10e5
2015/08/21 22:45:43 packer-post-processor-vsphere: reflect.Value.Call(0x3a1da0, 0x3dde90, 0x13, 0xc2080a1f28, 0x3, 0x3, 0x0, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/reflect/value.go:296 +0xbc
2015/08/21 22:45:43 packer-post-processor-vsphere: net/rpc.(*service).call(0xc208039c00, 0xc208039bc0, 0xc20800aa68, 0xc20807c500, 0xc2080fe280, 0x328820, 0xc2080dc088, 0xca, 0x307900, 0xc2080dc0b0, ...)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/net/rpc/server.go:382 +0x1f7
2015/08/21 22:45:43 packer-post-processor-vsphere: created by net/rpc.(*Server).ServeCodec
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/net/rpc/server.go:476 +0x44a
2015/08/21 22:45:43 packer-post-processor-vsphere:
2015/08/21 22:45:43 packer-post-processor-vsphere: goroutine 81 [select, 34 minutes]:
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/hashicorp/yamux.(*Stream).Read(0xc2080ec140, 0xc208143000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/yamux/stream.go:125 +0x3ab
2015/08/21 22:45:43 packer-post-processor-vsphere: bufio.(*Reader).fill(0xc2080ea540)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/bufio/bufio.go:97 +0x1ce
2015/08/21 22:45:43 packer-post-processor-vsphere: bufio.(*Reader).ReadByte(0xc2080ea540, 0x14c965, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/bufio/bufio.go:196 +0x86
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/hashicorp/go-msgpack/codec.(*ioDecReader).readn1(0xc20803be30, 0x308560)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/go-msgpack/codec/decode.go:90 +0x45
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/hashicorp/go-msgpack/codec.(*msgpackDecDriver).initReadNext(0xc2081160c0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/go-msgpack/codec/msgpack.go:540 +0x48
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/hashicorp/go-msgpack/codec.(*Decoder).decode(0xc2080ea5a0, 0x30cd00, 0xc20811a150)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/go-msgpack/codec/decode.go:635 +0x59
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/hashicorp/go-msgpack/codec.(*Decoder).Decode(0xc2080ea5a0, 0x30cd00, 0xc20811a150, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/go-msgpack/codec/decode.go:630 +0x7f
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/hashicorp/go-msgpack/codec.(*rpcCodec).read(0xc2080f22c0, 0x30cd00, 0xc20811a150, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/go-msgpack/codec/rpc.go:88 +0xd9
2015/08/21 22:45:43 packer-post-processor-vsphere: github.com/hashicorp/go-msgpack/codec.(*goRpcCodec).ReadResponseHeader(0xc2080f22c0, 0xc20811a150, 0x0, 0x0)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/private/tmp/packer20150813-68003-e1u33m/src/github.com/hashicorp/go-msgpack/codec/rpc.go:123 +0x61
2015/08/21 22:45:43 packer-post-processor-vsphere: net/rpc.(*Client).input(0xc2080ea600)
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/net/rpc/client.go:109 +0xb6
2015/08/21 22:45:43 packer-post-processor-vsphere: created by net/rpc.NewClientWithCodec
2015/08/21 22:45:43 packer-post-processor-vsphere: 	/usr/local/Cellar/go/1.4.2/libexec/src/net/rpc/client.go:201 +0xd0
2015/08/21 22:45:45 Deleting original artifact for build 'vmware-vmx'
2015/08/21 22:45:45 /usr/local/bin/packer-post-processor-vsphere: plugin process exited
2015/08/21 22:45:45 ui error: Build 'vmware-vmx' errored: 1 error(s) occurred:

* Post-processor failed: unexpected EOF
2015/08/21 22:45:45 Builds completed. Waiting on interrupt barrier...
2015/08/21 22:45:45 machine readable: error-count []string{"1"}
2015/08/21 22:45:45 ui error:
==> Some builds didn't complete successfully and had errors:
2015/08/21 22:45:45 machine readable: vmware-vmx,error []string{"1 error(s) occurred:\n\n* Post-processor failed: unexpected EOF"}
2015/08/21 22:45:45 ui error: --> vmware-vmx: 1 error(s) occurred:

* Post-processor failed: unexpected EOF
2015/08/21 22:45:45 ui:
==> Builds finished but no artifacts were created.
2015/08/21 22:45:45 waiting for all plugin processes to complete...
Build 'vmware-vmx' errored: 1 error(s) occurred:

* Post-processor failed: unexpected EOF

==> Some builds didn't complete successfully and had errors:
--> vmware-vmx: 1 error(s) occurred:

* Post-processor failed: unexpected EOF

==> Builds finished but no artifacts were created.
2015/08/21 22:45:45 /usr/local/bin/packer-builder-vmware-vmx: plugin process exited
benjaminbailey-17:packer devops$

**Comment by cbednarski** [comment]
Thanks for the report! I think this is the same as another issue that was reported last week: https://github.com/mitchellh/packer/issues/2595

Can you tell me how much memory you have on the machine running the packer build (not the vsphere machine)? The log makes it look like it crashed after trying to allocate 64gb of ram, which is kinda nuts.

There's definitely a bug here but I have not had time to investigate yet. Sorry about the inconvenience!

**Comment by cbednarski** [comment]
@ubailb1 I have added a gist with your log to #2595. Since this is a duplicate let's move the conversation to the other issue so I can keep it in one place. I'm going to close this issue for now. Thanks!
